Automattic/jetpack
# Paid stats: sync wpcom site features

Related to https://github.com/Automattic/red-team/issues/297

## Proposed changes:

- Moves Paid stats from Premium plans to Personal plans.
- Syncs with the changes in D167647-code (now merged).
- There should be no changes to Jetpack or even Jetpack stats with this change as this file is not leveraged by Jetpack stats (odyssey), this PR is just to keep the file in sync with wpcom.

### Other information:

P2: p1HpG7-v7f-p2

- [x] Have you written new tests for your changes, if applicable? See D167647-code

## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

- Should be a no op, smoke test on a woa dev site with the wpcomsh plugin set to this branch using the Jetpack beta plugin
- See also: https://github.com/Automattic/wp-calypso/pull/97041 and D167647-code for testing plans around these changes (deployed).
